Readme.md

```diff
@@ -1,6 +1,6 @@
-# BaSP 2023 - [app name]
+# BaSP 2023 - Juvi MegaRocket Server
```

Comment:
The name of the project is "Megarocket", not "Juvi Megarocket". We write juvi in some places to identify our team, not the project.